app/views/handlers/individual_response.py

```diff
@@ -160,59 +160,56 @@ def _publish_fulfilment_request(self, mobile_number=None):
         )
 
     def handle_get(self):
+        return render_template(
+            template="individual_response/interstitial",
+            language=self._language,
+            previous_location_url=self._get_previous_location_url(),
+            next_location_url=self._get_next_location_url(),
+            page_title=self.page_title(
+                lazy_gettext("Cannot answer questions for others in your household")
+            ),
+        )
+
+    def _get_next_location_url(self):
+        list_model = self._questionnaire_store.list_store[self._list_name]
+
+        if self._list_item_id:
+            return url_for(
+                ".individual_response_how",
+                list_item_id=self._list_item_id,
+                journey=self._request_args.get("journey"),
+            )
+        elif len(list_model.non_primary_people) == 1:
+            return url_for(
+                ".individual_response_how",
+                list_item_id=list_model.non_primary_people[0],
+                journey="hub",
+            )
+        else:
+            return url_for(".individual_response_who", journey="hub")
+
+    def _get_previous_location_url(self):
         individual_section_first_block_id = self._schema.get_first_block_id_for_section(
             self.individual_section_id
         )
```

Comment:
As this is only used in the second conditional, these lines should be moved there.